Source/Scene/PostProcessorScene.js

```diff
@@ -20,23 +20,247 @@ define([
      */
     function PostProcessorScene() {
         this.blackAndWhiteStage = createBlackAndWhiteStage();
+        this.blackAndWhiteStage.show = false;
         this.brightnessStage = createBrightnessStage();
+        this.brightnessStage.show = false;
         this.eightBitStage = createEightBitStage();
+        this.eightBitStage.show = false;
         this.compositeTextureStage = createCompositeTextureStage();
         this.compositeTextureStage.show = false;
+        this.LensFlareStage = createLensFlareStage();
 
         var stages = [
             this.blackAndWhiteStage,
             this.brightnessStage,
             this.eightBitStage,
-            this.compositeTextureStage
+            this.compositeTextureStage,
+            this.LensFlareStage
         ];
 
         this._postProcessor = new PostProcessor({
             stages : stages
         });
     }
 
+    function createLensFlareStage() {
+        var url_Dirt = buildModuleUrl('Assets/Textures/LensFlare/DirtMask.jpg');
+        var url_Star = buildModuleUrl('Assets/Textures/LensFlare/StarBurst.jpg');
+
+        var uniformValues = {
+
+            DirtTexture: url_Dirt,
+            StarTexture: url_Star,
+
+            Intensity: 2.0,
+
+            Distortion: 5.0,
+            NumofGhosts: 4,
+            GhostDispersal: 0.4,
+
+            HaloWidth: 0.4,
+
+            EarthRadius: 0.0
+        };
+
+        var LensFlareShader =
+            'uniform sampler2D u_colorTexture; \n' +
+            'uniform sampler2D u_DirtTexture; \n' +
+            'uniform sampler2D u_StarTexture; \n' +
+
+            'uniform float u_Distortion; \n' +
```

Comment:
It does support `vec4, should work by sending a `Cartesian4` to the uniform values area.